### PR TITLE
swarm.Address - ByteString()

### PR DESCRIPTION
**Pull request by pradovic**

### CODE REVIEW

File: pkg/swarm/swarm.go [hunk 1]
```diff
@@ -61,6 +61,11 @@ func (a Address) Bytes() []byte {
 	return a.b
 }
 
+// ByteString returns raw Address string without decoding.
```
Comment (janos) [inline pkg/swarm/swarm.go]: Should be `encoding` instead of `decoding` in the comment, as tis function encodes?

Comment (pradovic) [reply pkg/swarm/swarm.go]: Yes, sorry 🙈

Comment (pradovic) [reply pkg/swarm/swarm.go]: Introduced a typo to fix a typo :trollface: 😂

**Comment by pradovic** [comment]
@acud It's what we talked about, to avoid encoding, when using it as a key in a map? Does it make sense?

**Pull request merged**